Makefile has bad path for `pre-commit`

### Description

Getting the following error when running `make install`:

```
make install
Resolved 110 packages in 30ms
Audited 110 packages in 0.62ms
Installed dependencies
Installing pre-commit git hooks...
/bin/sh: .venv/pre-commit: No such file or directory
make[1]: *** [install_hooks] Error 127
make: *** [install] Error 2
```

This is because the path for pre-commit in the Makefile is listed as `VENV_DIR/pre-commit` where it should be `VENV_BIN/pre-commit`

### Steps to Reproduce

1. Run `make install`
2. See error above.


### Expected Behavior

Should run without error.

--- a/Makefile
+++ b/Makefile
@@ -100,7 +100,7 @@ install: check_env venv ## Install dependencies from the lockfile for the specif
 install_hooks: ## Install 'pre-commit' git hooks, only for local.
 	@if [ $(ENV) == local ]; then \
 		echo "Installing pre-commit git hooks..."; \
-		$(VENV_DIR)/pre-commit install; \
+		$(VENV_BIN)/pre-commit install; \
 	else \
 		echo "Not a git repository. Skipping pre-commit hooks installation."; \
 	fi

